.github/workflows/content-mapper-conformance.yml

```
@@ -0,0 +1,97 @@
+          persist-credentials: false
+      - name: Setup Go
+        uses: actions/setup-go@4dc6199c7b1a012772edbd06daecab0f50c9053c # v6.1.0
+        with:
+          go-version-file: typescript-go-content-mapper/go.mod
+          cache-dependency-path: typescript-go-content-mapper/go.sum
+      - name: Setup Rust
+        uses: dtolnay/rust-toolchain@29eef336d9b2848a0b548edc03f92a220660cdb8 # stable
+      - uses: wild-linker/action@0bbbfa5df4380cab8e63cb8505a1ce65e1d10203 # v0.9.0
+        with:
+          wild-version: "0.9.0"
+      - uses: ./.github/actions/setup-rust-sticky-cache
```

actionlint (1.7.12)

[error] missing input "cache-key-suffix" which is required by action "Setup Rust sticky cache" defined at "./.github/actions/setup-rust-sticky-cache". all required inputs are "cache-key-suffix", "key"

(action)